api/address/routes.go

```diff
@@ -21,6 +22,7 @@ const (
 	getKeyPath      = "/:address/key/:key"
 	getESDTTokens   = "/:address/esdt"
 	getESDTBalance  = "/:address/esdt/:tokenIdentifier"
+	getESDTNFTData  = "/:address/esdtnft/:tokenIdentifier/:nonce"
```

Comment:
I would suggest using `/:address/esdtnft/:tokenIdentifier/nonce/:nonce` to be consistent with other endpoints

Reply:
done